sui/src/bin/node.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
I wonder if we can have a general purpose `sui-node` binary that can handle launching either a full-node or a validator instead of needing a different binary for every node type we want to have.

Reply:
I suggested doing that earlier, but I believe @oxade had some objections. I think it would speed up docker builds significantly since we currently re-link several 99% identical binaries.